### ISSUE TITLE
Final Edits

### ISSUE THREAD

**Issue by cleebp** [cleebp/csc-510-group-g#6]
Hey all, I'll be reading through the paper multiple times today and just making general edits/cleaning up grammar if I notice anything.

If we have anything discussion wise we want to bring up to the group we can post those here!

**Comment by cleebp** [comment]
I'll start with one question, in regards to mentioning "Clipit" throughout the paper, should we just mention it in regular plain text, Clipit, or should we italicize it, _Clipit_?

If anyone has some thoughts one way or the other feel free to chime in, if no one cares we can just leave it plain text!

**Comment by reachspirit** [comment]
I think the font doesn't matter as long as they are in consistent with each
other. Well, I would prefer italic : )

Ran

**Comment by arjunmadan** [comment]
I think italics would be nice.

**Comment by cleebp** [comment]
Alright changing it to Italics throughout :D

Unfortunately when you try to italicize it in the subtitle it makes the font super tiny so that will be the only instance of it where it won't be italicized.

**Comment by cleebp** [comment]
Also I fixed the section referencing, essentially we need to put \label{sec:sectionLabel} after each section declaration, and then you can simply say ~\ref{sectionLabel} from anywhere, pretty cool!

**Comment by cleebp** [comment]
Just finished up my first round of edits, I changed up a lot of little things here and there mostly to make sure our grammar was consistent throughout. I also spent a lot of time on the project features section so that it flows better with the rest of the paper.

I just reread it again and it looks super good and I'm happy with it, but I will look at it again in a couple hours to see if I can find any more grammar problems, etc. At that point if we don't see anything else I'll push the pdf to our main repo and add the bit.ly url to the submission page - we will still have until midnight though to change anything if anyone finds something wrong!
